gossip/src/cluster_info.rs

```diff
@@ -3399,6 +3412,8 @@ mod tests {
             sync::Arc,
         },
     };
+    const DEFAULT_NUM_QUIC_ENDPOINTS: NonZeroUsize =
```

Comment:
nit: personally I'd rather use new(...).unwrap() and avoid unsafe

Reply:
The unwrap cause clippy issue -- I will leave this as is.